### ISSUE TITLE
the train.lua stop print informatation when the input file is bigger than 500MB?

### ISSUE THREAD

**Issue by esrrhs** [crisbal/docker-torch-rnn#17]
Hi, my input file is very big, about 500MB. and the train.lua stop print information when run for a while. the last info it's print is 

Epoch 1.02 / 50, i = 994 / 3145800, loss = 4.990288
Epoch 1.02 / 50, i = 995 / 3145800, loss = 5.104537
Epoch 1.02 / 50, i = 996 / 3145800, loss = 4.961758
Epoch 1.02 / 50, i = 997 / 3145800, loss = 4.969568
Epoch 1.02 / 50, i = 998 / 3145800, loss = 5.046015
Epoch 1.02 / 50, i = 999 / 3145800, loss = 4.955519
Epoch 1.02 / 50, i = 1000 / 3145800, loss = 4.886581

and the process CPU run at 100%
PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                                                                                
31256 root      20   0 2333920   2.0g   1916 R 108.0 54.5 172:40.70 luajit 

is this normal? should I just wait?
